.github/workflows/release.yml

```diff
@@ -1,29 +1,79 @@
-# This Workflow performs a full release of chectl
-name: Release chectl
+#
+#  Copyright (c) 2012-2020 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+name: Build and Deploy release artifacts ⌛
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'The version that is going to be released. Should be in format 7.y.z'
-        required: true
-        default: '7.y.z'
+  # Trigger the workflow on push only for the master branch
+  push:
+    branches:
+      - '7*' # Push events to matching 7.* branches
+      - 'master'
 jobs:
-  build:
-    name: Create Release
-    runs-on: ubuntu-latest
+  build-and-deploy:
+    runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
         with:
-          fetch-depth: 0
-          token: ${{ secrets.ACTIONS_TOKEN }}
-      - uses: actions/setup-node@v2.1.2
+          persist-credentials: false
+      - uses: actions/setup-node@v1
         with:
-          node-version: 12.18.2
-      - name: Run make-release.sh script
+          node-version: '12'
+      - name: Generate tag utilities 📜
+        id: TAG_UTIL
         run: |
-          # since we're using Flavius' token below to generate PRs, should also make tags/branches/commits by same owner
-          # see https://github.com/che-incubator/chectl/settings/secrets/actions for the token used
-          git config --global user.name "Flavius Lacatusu"
-          git config --global user.email "flacatus@redhat.com"
-          export GITHUB_TOKEN=${{ secrets.ACTIONS_TOKEN }}
-          /bin/bash -x make-release.sh --version ${{ github.event.inputs.version }}
+          echo "::set-output name=gh_tag::$(date +'%Y%m%d%H%M%S')"
```

Comment:
what do these `::set-output ...` echos do? are they just informative console info?

Reply:
::set-output I'm setting the necessary tags and then I can use it in all part of job. Works like envs

Reply:
so `TAG_UTIL` somehow gets the variables ?

Reply:
Yeah like here.https://github.com/che-incubator/chectl/blob/remove_travis/.github/workflows/release.yml#L56
I found the docs today here: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions

Reply:
well, that's an odd syntax, but hey, if that's the best way to have global vars persist between steps, so be it.

Reply:
Just a FYI you can use syntax like to share env variables ( and not step output)

echo "PR_NUMBER=$pr_number" >> $GITHUB_ENV